Objects/cellobject.c

```diff
@@ -140,8 +140,19 @@ cell_get_contents(PyCellObject *op, void *closure)
     return op->ob_ref;
 }
 
+int
+cell_set_contents(PyCellObject *op, PyCellObject *obj)
+{
+    Py_INCREF(obj);
+    PyCellObject *tmp_op = (PyCellObject *)(op->ob_ref);
```

Comment:
This is what I originally had in my first commit, but running `make patchcheck` showed this warning:
`incompatible pointer types assigning to 'PyObject *' (aka 'struct _object *') from 'PyCellObject *'`

Which is why I rewrote with the current lines. What do you think?